### ISSUE TITLE
Amalthea-scheduler image guessing fails on EKS

### ISSUE THREAD

**Issue by aledegano** [SwissDataScienceCenter/amalthea#699]
The guessing mechanism for the version of the Amalthea scheduler fails on EKS because when it queries Kubernetes for the version gets something like: `1.29.6-eks-db838b0` and then tries to launch the image `registry.k8s.io/kube-scheduler:v1.29.6-eks-db838b0` which obviously does not exist.

It would probably be enough to check the returned string from the query and remove anything past the last digit.

**Comment by olevski** [comment]
This is in the helm chart templates, right @aledegano ?

**Comment by aledegano** [comment]
Yes, I believe so. I might give it a try, but I wanted to open the issue to not forget of the problem.
